lib/resources/r4/observation.yaml

```diff
@@ -55,6 +306,170 @@ fields:
     - display: Interpretation Result
       system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/52
       info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-52-interpretation-result
+  example: |
+    {
+      "interpretation": [
+        {
+          "coding": [
+            {
+              "system": 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/52',
+              "code": '211',
+              "display": 'LOW',
+              "userSelected": true
+            }
+          ],
+          "text": 'LOW'
+        }
+      ]
+    }
+  example2: |
+    {
+      "interpretation": [
+        {
+          "coding": [
+            {
+              {
+                "system": 'http://terminology.hl7.org/CodeSystem/v3-ObservationInterpretation',
+                "code": 'L',
+                "display": 'Low'
+              }
+            ],
+            "text": 'LOW'
+          }
+        ]
+      }
+  note: |
+    <ul>
+      <li> Only one interpretation may be supplied. </li>
+    </ul>
+  url: https://hl7.org/fhir/R4/observation-definitions.html#Observation.interpretation
+
+- name: note
+  required: 'No'
+  type: List of Annotation
+  description: Comments about the observation
+  action: create
+  example: |
+    {
+      "note": [
+        {
+          "authorReference": {
+            "reference": 'Practitioner/62955753'
+          },
+          "time": '2018-08-17T09:55:42.000Z',
+          "text": 'Charted at Incorrect Time'
+        }
+      ]
+    }
+  url: https://hl7.org/fhir/r4/observation-definitions.html#Observation.note
+
+- name: referenceRange
+  required: 'No'
+  type: List of BackboneElement
+  description: Provides guidance for interpretating the observation.
+  action: create
+  example: |
+    {
+      "referenceRange": [
+        {
+          "low": {
+            "value": 35.2,
+            "unit": 'degC',
+            "system": 'http://unitsofmeasure.org',
+            "code": 'Cel'
+          },
+          "high": {
+            "value": 36.7,
+            "unit": 'degC',
+            "system": 'http://unitsofmeasure.org',
+            "code": 'Cel'
+          },
+          "type": {
+            "coding": [
+              {
+                "system": 'http://terminology.hl7.org/CodeSystem/referencerange-meaning',
+                "code": 'normal',
+                "display": 'Normal Range'
+              }
+            ],
+            "text": 'Normal Range'
+          }
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li> Only one referenceRange is supported </li>
+      <li> ReferenceRange.text is not supported when referenceRange.high and/or referenceRange.low are provided </li>
```

Comment:
field names should be quoted as in other note sections.

Reply:
updated: https://github.com/cerner/fhir.cerner.com/commit/a3c496078c9f8365f3c8e5b8e0e7d18a4d98b3ae#diff-53f5b7c6241dd914461705ddc9523c7eR402